client/src/component/Title/Title.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import 'font-awesome/css/font-awesome.min.css'
+import './Title.css'
+
+const Title = () => {
+    return (
+        <article className="Title" id="Header">
+            <section className="container">
+                <header>HACKMERCED V</header>
+                <div>20/20/2020</div>
+
+                <div class="box">
```

Comment:
```suggestion
                <div className="box">
```

In React they removed the word `class` in the html and changed it with `className`